yaksh/management/commands/create_moderator.py

```diff
@@ -35,11 +36,14 @@ def handle(self, *args, **options):
                         uname)
                     )
                 if user in group.user_set.all():
-                    self.stdout.write('User "{0}" is '
+                    self.stdout.write(self.style.WARNING('User "{0}" is '
                                       'already a Moderator'.format(uname)
-                                      )
+                                      ))
                 else:
                     group.user_set.add(user)
```

Comment:
You can remove this line. Since setting ```is_moderator``` to ```True``` will now add the user to the Moderator group once you save the profile.

Please refer to the ```save``` and ```_toggle_moderator_group``` methods of the ```Profile``` class in ```yaksh/models.py```

Reply:
Done.